### PR TITLE
fix: Missing padding on Omnichannel contacts `Contextualbar` loading state

### LINKED ISSUE #29181: No padding to the Skeleton placeHolder in the omnichanel contact centre
Description:

Missing padding for the skeleton placeholder in the omnichanel contact centre

### Steps to reproduce:

1. Click on the omnichanel directory icon in the main sidebar column

 Actual behavior:
Error_1

 Expected behavior:
there should be padding given and the skeleton placeholder should be properly aligned

### PR DESCRIPTION
**Pull request by Kishn0109**
## Issue(s)

Closes #29181 
Closes #30031
## Further comments

I have to create a different branch with the same changes as in my preview MR I did this because I was facing a problem to signing the CLA agreement and I tried to match my commiter name with author name in between that my previews PR hade so many mistakes and result closed by the rocket chatbot. Please consider it and I apologize for the inconvenience

### CODE REVIEW

**Comment by Kishn0109** [comment]
Hi, @KevLehman I resolved this issue a week ago still it is in status QA needed, if there are some problems please let me I will be happy to fix them thank you

**Review by KevLehman — APPROVED** [review]

**Comment by Kishn0109** [comment]
Hi team can this MR get reviewed? I Merge branch 'develop' into fix/skeleton-loading by mistake because of that   the previous review got dismissed

**Comment by KevLehman** [comment]
Let me try to move this forward on Monday :)

**Review by dougfabris — CHANGES_REQUESTED** [review]
@Kishn0109 Thanks for your contribution, could you replace the `Box` usage with `ContextualbarContent` as a wrapper to the `FormSkeleton`?

File: apps/meteor/client/views/omnichannel/directory/contacts/contextualBar/ContactEditWithData.js [hunk 1]
```diff
@@ -16,7 +16,11 @@ function ContactEditWithData({ id, close }) {
 	} = useEndpointData('/v1/omnichannel/contact', { params: useMemo(() => ({ contactId: id }), [id]) });
 
 	if ([state].includes(AsyncStatePhase.LOADING)) {
-		return <FormSkeleton />;
+		return (
+			<Box p={'x24'}>
+				<FormSkeleton />;
```
Comment (aleksandernsilva) [inline apps/meteor/client/views/omnichannel/directory/contacts/contextualBar/ContactEditWithData.js:21]:
```suggestion
				<FormSkeleton />
```

**Review by aleksandernsilva — DISMISSED** [review]

**Comment by Kishn0109** [comment]
I really appreciate your suggestions! I have made the changes you asked for based on your feedback. Instead of using the Box component, I have used ContextualbarContent as a wrapper around FormSkeleton, as you suggested.

File: apps/meteor/client/views/omnichannel/directory/contacts/contextualBar/ContactEditWithData.js [hunk 2]
```diff
@@ -16,7 +16,11 @@ function ContactEditWithData({ id, close }) {
 	} = useEndpointData('/v1/omnichannel/contact', { params: useMemo(() => ({ contactId: id }), [id]) });
 
 	if ([state].includes(AsyncStatePhase.LOADING)) {
-		return <FormSkeleton />;
+		return (
+			<ContextualbarContent p={'x24'}>
```
Comment (KevLehman) [inline apps/meteor/client/views/omnichannel/directory/contacts/contextualBar/ContactEditWithData.js:20]:
```suggestion
			<ContextualbarContent p='x24'>
```

File: apps/meteor/client/views/omnichannel/directory/contacts/contextualBar/ContactNewEdit.tsx [hunk 3]
```diff
@@ -178,7 +178,11 @@ const ContactNewEdit = ({ id, data, close }: ContactNewEditProps): ReactElement
 	};
 
 	if (isLoadingCustomFields) {
-		return <FormSkeleton />;
+		return (
+			<ContextualbarContent p={'x24'}>
```
Comment (KevLehman) [inline apps/meteor/client/views/omnichannel/directory/contacts/contextualBar/ContactNewEdit.tsx:182]:
```suggestion
			<ContextualbarContent p='x24'>
```

**Review by KevLehman — CHANGES_REQUESTED** [review]

**Review by KevLehman — DISMISSED** [review]

File: apps/meteor/client/views/omnichannel/directory/contacts/contextualBar/ContactEditWithData.js [hunk 4]
```diff
@@ -16,7 +16,11 @@ function ContactEditWithData({ id, close }) {
 	} = useEndpointData('/v1/omnichannel/contact', { params: useMemo(() => ({ contactId: id }), [id]) });
 
 	if ([state].includes(AsyncStatePhase.LOADING)) {
-		return <FormSkeleton />;
+		return (
+			<ContextualbarContent p='x24'>
```
Comment (dougfabris) [inline apps/meteor/client/views/omnichannel/directory/contacts/contextualBar/ContactEditWithData.js:20]:
```suggestion
			<ContextualbarContent>
```

File: apps/meteor/client/views/omnichannel/directory/contacts/contextualBar/ContactNewEdit.tsx [hunk 5]
```diff
@@ -178,7 +178,11 @@ const ContactNewEdit = ({ id, data, close }: ContactNewEditProps): ReactElement
 	};
 
 	if (isLoadingCustomFields) {
-		return <FormSkeleton />;
+		return (
+			<ContextualbarContent p='x24'>
```
Comment (dougfabris) [inline apps/meteor/client/views/omnichannel/directory/contacts/contextualBar/ContactNewEdit.tsx:182]:
```suggestion
			<ContextualbarContent>
```

**Review by dougfabris — CHANGES_REQUESTED** [review]

**Review by dougfabris — APPROVED** [review]

**Pull request merged**